subsys/bluetooth/controller/ll_sw/nordic/lll/lll_adv.c

```diff
@@ -1034,34 +1035,38 @@ static int prepare_cb(struct lll_prepare_param *p)
 
 #if defined(CONFIG_BT_CTLR_XTAL_ADVANCED) && \
 	(EVENT_OVERHEAD_PREEMPT_US <= EVENT_OVERHEAD_PREEMPT_MIN_US)
+	uint32_t overhead;
+
+	overhead = lll_preempt_calc(ull, (TICKER_ID_ADV_BASE +
+					  ull_adv_lll_handle_get(lll)),
+				    ticks_at_event);
```

Comment:
```suggestion
	overhead = lll_preempt_calc(ull, (TICKER_ID_ADV_BASE + ull_adv_lll_handle_get(lll)),
				    ticks_at_event);
```

I would still suggest to switch to using 100 column width as per the code style guide :)